### DISCUSSION TITLE
The root pane of GameSubScene should be mouse transparent?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1240] [Q&A] [answered]
Hi:

Is it a good idea to set the root pane mouse transparent?
Usually developers should not access the root pane of game scene
but if we overlap multiple game sub scenes, then some times we want the UI buttons which is not shadowed by the scenes in front
especially for those transparent background color
so is it a good idea to make it mouse transparent by default?

Thanks

**Comment by AlmasB** [comment, marked as answer]
If the functionality can be overridden / implemented by user code, I'm inclined to leave it as is.
